### PR TITLE
added additional inputmode attr to relevant form input fields

### PR DESCRIPTION
**Pull request by dafunction**
missed quite a few numerical form inputs in [PR#3575](https://github.com/btcpayserver/btcpayserver/pull/3575)

i believe i got them all this time :)

### CODE REVIEW

File: BTCPayServer/Views/UIApps/TemplateEditor.cshtml [hunk 1]
```diff
@@ -69,7 +69,7 @@
                                     
                                     <label class="form-label">&nbsp</label>
                                     <input class="form-control mb-2"
-                                           inputmode="numeric"
```
Comment (bolatovumar) [inline BTCPayServer/Views/UIApps/TemplateEditor.cshtml:72]: Does specifying `inputmode` on input of type `number` do anything? This type of input should bring up the numerical keyboard even without `inputmode` specified, no?

Comment (dennisreimann) [reply BTCPayServer/Views/UIApps/TemplateEditor.cshtml:72]: @bolatovumar See the notes in #3575 :)

File: BTCPayServer/Views/UIStores/CheckoutAppearance.cshtml [hunk 2]
```diff
@@ -36,7 +36,7 @@
                                     </select>
                                 </td>
                                 <td class="border-0">
-                                    <input placeholder="6.15 USD" asp-for="PaymentMethodCriteria[index].Value" class="form-control my-0" style="max-width: 20ch;" />
+                                    <input placeholder="6.15 USD" inputmode="decimal" asp-for="PaymentMethodCriteria[index].Value" class="form-control my-0" style="max-width: 20ch;" />
```
Comment (dennisreimann) [inline BTCPayServer/Views/UIStores/CheckoutAppearance.cshtml:39]: This one's a `text` input as it also contains the currency code.

Comment (dafunction) [reply BTCPayServer/Views/UIStores/CheckoutAppearance.cshtml:39]: ahh good call - i'll remove the attribute from that element in a separate commit.

**Review by dennisreimann — APPROVED** [review]

**Pull request merged**